Provider.php

```diff
@@ -57,11 +60,11 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'       => $user['guid'],
+            'id'       => $user['sub'],
             'nickname' => $user['nickname'],
```

Comment:
nickname was not present when I tried this out, I'd recommend:

```php
'nickname' => $user['nickname'] ?? null,
```

Reply:
what will the nickname  value be when nickname was not present ? 
We can change the code as you recommend

Reply:
Also,  if $user['nickname'] not present and the value already is null, 
then  $user['nickname'] ?? null  will just equal to $user['nickname']?